### PR TITLE
popup corrected

### LINKED ISSUE #664: On click different image popup.
### In Memories from FOSSASIA Events section different image popup when clicked on an image for it's preview.
As screenshot attached image with Red line have this issue.

fossasia issue

@mariobehling is this a valid issue?

**If so,I would like to work on it.**

### PR DESCRIPTION
**Pull request by Sohan022**
On clicking, different images popup in memories section of FOSSASIA corrected

fixed: #664

preview link: https://sohan022.github.io/fossasia.org/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -194,7 +194,7 @@
 
 						<div class="copy-text-box">
 							<div class="copy-text-bottom">
-								<span>© Copyright 2009-2020 Creative Commons By License, FOSSASIA</span>
+								<span>© Copyright 2009-2019 Creative Commons By License, FOSSASIA</span>
```
Comment (Chaitanyassr) [inline index.html:197]: Why change 2020 to 2019?

File: index.html [hunk 2]
```diff
@@ -350,9 +350,9 @@ <h3>Events and Meetups<br></h3>
 								<div class="info-box">
 									<img alt="Venue" src="img/16704948658_0dff7bd62c_z.jpg">
 									<h3>Coding Programs and Labs<br></h3>
-									<p><div>At FOSSASIA Labs we are collecting ideas for new features of our applications and entirely new projects. In our coding contests and development and mentorship programs we support new developers to join the Open Tech community. The annual Codeheat contest offers many prizes and the winners fly to the FOSSASIA Summit in Singapore.</div><div><br><br><br></div></p>
+									<p><div>At FOSSASIA Labs we are collecting ideas for new features of our applications and entirely new projects. In our coding contests and development and mentorship programs we support new developers to join the Open Tech community. The annual Codeheat contest offers many prizes and the winners fly to the FOSSASIA Summit in Singapore. We also cooperate with partners in programs such as Google Summer of Code.</div><div><br><br><br></div></p>
```
Comment (Chaitanyassr) [inline index.html:353]: Why adding extra text?

File: index.html [hunk 3]
```diff
@@ -1317,7 +1317,7 @@ <h3>High School Student Program</h3>
 								<div class="info-box">
 									<img alt="Advanced Developer Program" src="img/advanced-dev-program.jpg">
 									<h3>Advanced Developer Program</h3>
-									<p>The 'Advanced Developer Program' is intended for developers who have previously participated successfully in an intensive FOSSASIA program.<div><br></div><div><br></div></p>
+									<p>The 'Advanced Developer Program' is intended for developers who have previously participated successfully in an intensive FOSSASIA program such as GSoC.<div><br></div><div><br></div></p>
```
Comment (Chaitanyassr) [inline index.html:1320]: Why adding extra text here?

**Review by Chaitanyassr — CHANGES_REQUESTED** [review]

**Comment by Chaitanyassr** [comment]
> On clicking, different images popup in memories section of FOSSASIA corrected
> 
> fixed: #664
> 
> preview link: https://sohan022.github.io/fossasia.org/

Is your branch is updated, if not than please make sure ones.

**Comment by Sohan022** [comment]
My branch is not updated. So, I make a new pull request.

**Comment by Chaitanyassr** [comment]
> My branch is not updated. So, I make a new pull request.

just add an upstream branch and pull or merge it.

**Pull request closed without merging**